qiskit_sphinx_theme/__init__.py

```diff
@@ -13,6 +13,7 @@
 """Pytorch Sphinx theme."""
 
 from pathlib import Path
+from typing import Any, Dict
```

Comment:
You don't need to import `Dict`. Our repo uses Python 3.8+, so you can use the new builtin `dict` (lowercase)